modules/core/src/test/resources/catalogs/landsat8-layers/collection-layer-ca-v1.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
This is a first pass to generate such a layer, in terms of this PR I'll make it completely valid and working.